fastmri_recon/data/utils/multicoil/smap_extract.py

```diff
@@ -1,6 +1,6 @@
 import tensorflow as tf
 from tensorflow.python.ops.signal.fft_ops import ifft2d, ifftshift, fftshift
-
+from math import pi
```

Comment:
I think `math` is a builtin library in Python so it should probably go above in the imports.